package.json

```diff
@@ -42,17 +43,19 @@
     "@semantic-release/npm": "^7.0.3",
     "@semantic-release/release-notes-generator": "^9.0.0",
     "@types/eslint": "^6.1.8",
-    "@types/estree": "0.0.42",
+    "@types/jest": "^25.0.0",
     "@types/node": "^13.7.1",
-    "@typescript-eslint/eslint-plugin": "^2.19.2",
-    "@typescript-eslint/parser": "^2.19.2",
+    "@typescript-eslint/eslint-plugin": "^2.34.0",
```

Comment:
suggest removing the peer dep on the plugin, as this plugin doesn't need it at all
```suggestion
```

Reply:
Good catch. Removed the peer dep for eslint as well since that's not used either.

Reply:
the peer dep on ESLint is expected. One of the use cases for peer deps is declaring modules that will _run_ this module.

Reply:
Got it. Added that back, accepting 6 or 7.